libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus-dv/defaults.parm

```diff
@@ -1,630 +1,455 @@
-SERIAL_PASSTIMO 15
-SERVO10_FUNCTION 34
-SERVO10_MAX 2000
-SERVO10_MIN 1000
-SERVO10_REVERSED 0
-SERVO10_TRIM 1000
-SERVO11_FUNCTION 35
-SERVO11_MAX 2000
-SERVO11_MIN 1000
-SERVO11_REVERSED 0
-SERVO11_TRIM 1000
-SERVO12_FUNCTION 36
-SERVO12_MAX 2000
-SERVO12_MIN 1000
-SERVO12_REVERSED 0
-SERVO12_TRIM 1000
-SERVO13_FUNCTION 0
-SERVO13_MAX 1900
-SERVO13_MIN 1100
-SERVO13_REVERSED 0
-SERVO13_TRIM 1500
-SERVO14_FUNCTION 0
-SERVO14_MAX 1900
-SERVO14_MIN 1100
-SERVO14_REVERSED 0
-SERVO14_TRIM 1500
-SERVO15_FUNCTION 0
-SERVO15_MAX 1900
-SERVO15_MIN 1100
-SERVO15_REVERSED 0
-SERVO15_TRIM 1500
-SERVO16_FUNCTION 0
-SERVO16_MAX 1900
-SERVO16_MIN 1100
-SERVO16_REVERSED 0
-SERVO16_TRIM 1500
-SERVO1_FUNCTION 0
-SERVO1_MAX 2000
-SERVO1_MIN 995
-SERVO1_REVERSED 0
-SERVO1_TRIM 995
-SERVO2_FUNCTION 0
-SERVO2_MAX 2000
-SERVO2_MIN 995
-SERVO2_REVERSED 0
-SERVO2_TRIM 995
-SERVO3_FUNCTION 0
-SERVO3_MAX 2000
-SERVO3_MIN 995
-SERVO3_REVERSED 0
-SERVO3_TRIM 995
-SERVO4_FUNCTION 0
-SERVO4_MAX 2000
-SERVO4_MIN 995
-SERVO4_REVERSED 0
-SERVO4_TRIM 995
-SERVO5_FUNCTION 0
-SERVO5_MAX 1900
-SERVO5_MIN 1100
-SERVO5_REVERSED 0
-SERVO5_TRIM 1500
-SERVO6_FUNCTION 0
-SERVO6_MAX 1900
-SERVO6_MIN 1000
-SERVO6_REVERSED 0
-SERVO6_TRIM 1500
-SERVO7_FUNCTION 0
-SERVO7_MAX 1900
-SERVO7_MIN 1100
-SERVO7_REVERSED 0
-SERVO7_TRIM 1500
-SERVO8_FUNCTION 0
-SERVO8_MAX 1900
-SERVO8_MIN 1000
-SERVO8_REVERSED 0
-SERVO8_TRIM 1500
-SERVO9_FUNCTION 33
-SERVO9_MAX 2000
-SERVO9_MIN 1000
-SERVO9_REVERSED 0
-SERVO9_TRIM 1000
-
-SERVO_BLH_AUTO 1
-SERVO_BLH_BDMASK 3840
-SERVO_BLH_OTYPE 5
-SERVO_BLH_POLES 42
-SERVO_BLH_TRATE 100
-SERVO_DSHOT_RATE 1
-SERVO_FTW_POLES 42
-SERVO_RATE 400
-SERVO_SBUS_RATE 50
-
-SR0_ADSB 0
-SR0_EXTRA1 0
-SR0_EXTRA2 0
-SR0_EXTRA3 0
-SR0_EXT_STAT 0
-SR0_PARAMS 0
-SR0_POSITION 0
-SR0_RAW_CTRL 0
-SR0_RAW_SENS 0
-SR0_RC_CHAN 0
-SR1_ADSB 0
-SR1_EXTRA1 0
-SR1_EXTRA2 0
-SR1_EXTRA3 0
-SR1_EXT_STAT 0
-SR1_PARAMS 0
-SR1_POSITION 0
-SR1_RAW_CTRL 0
-SR1_RAW_SENS 0
-SR1_RC_CHAN 0
-SR2_ADSB 0
-SR2_EXTRA1 10
-SR2_EXTRA2 10
-SR2_EXTRA3 2
-SR2_EXT_STAT 2
-SR2_PARAMS 0
-SR2_POSITION 10
-SR2_RAW_CTRL 0
-SR2_RAW_SENS 2
-SR2_RC_CHAN 2
-SR3_ADSB 0
-SR3_EXTRA1 0
-SR3_EXTRA2 0
-SR3_EXTRA3 0
-SR3_EXT_STAT 0
-SR3_PARAMS 0
-SR3_POSITION 0
-SR3_RAW_CTRL 0
-SR3_RAW_SENS 0
-SR3_RC_CHAN 0
-SR4_ADSB 0
-SR4_EXTRA1 0
-SR4_EXTRA2 0
-SR4_EXTRA3 0
-SR4_EXT_STAT 0
-SR4_PARAMS 0
-SR4_POSITION 0
-SR4_RAW_CTRL 0
-SR4_RAW_SENS 0
-SR4_RC_CHAN 0
-SRTL_ACCURACY 2
-SRTL_OPTIONS 0
-SRTL_POINTS 500
-
-SUPER_SIMPLE 0
-SURFTRAK_MODE 0
-SURFTRAK_TC 1
-TCAL_ENABLED 0
-TELEM_DELAY 0
-TERRAIN_OFS_MAX 0
-THR_DZ 35
-TKOFF_RPM_MAX 0
-TKOFF_RPM_MIN 1200
-TKOFF_SLEW_TIME 2
-TKOFF_THR_MAX 0.9
-WPNAV_ACCEL 350
-WPNAV_ACCEL_C 250
-WPNAV_ACCEL_Z 150
-WPNAV_JERK 5
-WPNAV_RADIUS 100
-WPNAV_RFND_USE 0
-WPNAV_SPEED 600
-WPNAV_SPEED_DN 300
-WPNAV_SPEED_UP 500
-WPNAV_TER_MARGIN 10
-WP_NAVALT_MIN 0
-WP_YAW_BEHAVIOR 2
-WVANE_ENABLE 0
+@include ../CubeOrangePlus/defaults.parm
+
+AHRS_EKF_TYPE,3
+AHRS_GPS_USE,2
+AHRS_ORIENTATION,2
+AHRS_TRIM_X,0

```

Comment:
As far as I understand, those TRIM params are also subject to change during the accelerometer calibration procedure.

Should they instead of being defaulted here belong to the "preserve parameters" set to be defined [in this issue](https://github.com/Drone-Volt/ardupilot/issues/9)?